mongodbatlas/resource_mongodbatlas_project_ip_whitelist.go

```diff
@@ -243,15 +243,7 @@ func resourceMongoDBAtlasProjectIPWhitelistDelete(d *schema.ResourceData, meta i
 		}
 
 		if entry != nil {
-			_, err := conn.ProjectIPWhitelist.Delete(context.Background(), ids["project_id"], ids["entry"])
-			if err != nil {
-				if strings.Contains(fmt.Sprint(err), "500") ||
-					strings.Contains(fmt.Sprint(err), "Unexpected error") ||
-					strings.Contains(fmt.Sprint(err), "UNEXPECTED_ERROR") {
-					return resource.RetryableError(err)
-				}
-				return resource.NonRetryableError(fmt.Errorf(errorWhitelistDelete, err))
-			}
+			return resource.RetryableError(fmt.Errorf(errorWhitelistDelete, "Whitelist still exists"))
```

Comment:
Another bug?  Just want to get an idea of the related fixes to the actual resources.

Reply:
Yes, it happens sometimes the error about an ip whitelist still exists, when an ip exists it should retry not another delete because for some reason it can fail and it should have another get to verify that, that's why I deleted Delete and retry instead.

Reply:
Yeah, still think that's a timing issue.  Just note that in the PR description, for example - "Fixed: retryable error for IP Whitelist resource"